### DISCUSSION TITLE
How to solve this: "Unable to negotiate with x.x.x.x port 22: no matching host key type found. Their offer: ssh-rsa

### DISCUSSION THREAD

**Discussion by lca-usal** [cmderdev/cmder#2984] [Q&A]
Hi, I'm trying to connect with an old Cisco switch and I get this answer. Please Can you help me?

Thank you!!!

**Comment by daxgames** [comment]
Not sure that has anything to do with Cmder.  I would just tell you to use Google or your favorite search engine.
